language-html.md

```diff
@@ -16,20 +22,78 @@ style–guides about HTML.
 >
 ```
 
+#### Self-closing tags (void elements):
+
+In the HTML 5 specification the `/` character is valid but has no effect for void elements such as `<br />, <hr />, <img />, <input />`.
+So in our Rabbit team we decided not to close such void elements , i.e.
+```html
+<img class="my-class" id="void-element"
+    src="please-do-not-use-slash-to-close-this-element"
+>
```

Comment:
```
Elements written on multi-lines MUST have the closing bracket of the opening
tag __aligned with its opening bracket__: Because we’re using 4 spaces
indentation, the aligned attributes would be aligned with the following code,
which makes it harder to read.
```
It sounds reasonably for me.